pyfr/backends/openmp/blasext.py

```diff
@@ -55,23 +55,31 @@ def errest(self, x, y, z, *, norm):
         if x.traits != y.traits != z.traits:
             raise ValueError('Incompatible matrix types')
 
-        cnt = x.leaddim*x.nrow
-        dtype = x.dtype
+        nrow, ldim, dtype = x.traits
+        ncola, ncolb = x.ioshape[1:]
 
         # Render the reduction kernel template
-        src = self.backend.lookup.get_template('errest').render(norm=norm)
+        src = self.backend.lookup.get_template('errest').render(norm=norm,
+                                                                ncola=ncola)
+
+        # Array for the error estimate
+        error = np.zeros((1, ncola))
```

Comment:
We should not be using a backend matrix type for this.  A simple numpy array of the correct type will do the job.  Will need to pass it in to the kernel explicitly as as err.ctypes.data, however.